src/Microsoft.OpenApi.Readers/ParseNodes/PropertyNode.cs

```diff
@@ -63,7 +68,7 @@ internal class PropertyNode : ParseNode
                     catch (OpenApiException ex)
```

Comment:
>catch (OpenApiException ex) [](start = 20, length = 27)

do we need to add similar catch like line 44?

Reply:
It's not essential because OpenApiReaderException derives from OpenApiException now, but I will do it so that the pointer is correct.

---
In reply to: [194837784](https://github.com/Microsoft/OpenAPI.NET/pull/276#discussion_r194837784) [](ancestors = 194837784)